tcms/bugs/models.py

```diff
@@ -32,6 +32,11 @@ class Bug(models.Model, UrlMixin):
     executions = models.ManyToManyField('testruns.TestExecution', related_name='bugs')
     tags = models.ManyToManyField('management.Tag', related_name='bugs')
 
+    class Meta:
+        permissions = (
+            ('add_bugtag', 'can add bug tag'),
+        )
+
```

Comment:
There are 2 things going on in this PR. Should be 2 separate PRs or at least 2 separate commits.

There's a possible bug in how the tag permission is specified and handled and changes will affect more methods.